lib/manager/reducers/projects.js

```diff
@@ -23,7 +23,9 @@ export const defaultState = {
     // in the feed source table, this is a validation status filter
     filter: null,
     // This is used to sort both the Project List or the FeedSourceTable
-    searchText: null
+    searchText: null,
+    // Used by the label filtering dropdown
+    labelsFilterMode: 'any'
```

Comment:
Per https://github.com/ibi-group/datatools-ui/pull/693/files#r677627210, add default state for `filters: []`

Reply:
Feels much better. Thanks!